JavaScript/Timer.js

```diff
@@ -0,0 +1,63 @@
+//this code is written to operationalize a timer with 3 main functions of Start, Stop and Reset.
+class Clock {
+    /*fourth take */
+
+    constructor(displayElement) {
+        this.display = displayElement;
+        this.state = "initial";
+        this.seconds = 0;
+        this.interval = null;
+    }
+
+    start() {
+        if (this.state === "initial") {
+            this.interval = setInterval(() => this.timer(), 1000);
```

Comment:
using 1000 here makes your timer to start almost one second after you pushed `start`.

Reply:
I called `this.timer()` right before the 14th line. It starts counting immediately after clicking the start button. But after reseting the timer and want to start it again; it takes one second to get started.